Lib/ctypes/__init__.py

```diff
@@ -344,7 +344,8 @@ def __init__(self, name, mode=DEFAULT_MODE, handle=None,
                  use_errno=False,
                  use_last_error=False,
                  winmode=None):
-        self._name = name
+        if name:
+            name = _os.fspath(name)
```

Comment:
Why not just assign name here itself to self? You don't need to change it everywhere.

Reply:
Thank you, modified. Initially I wanted to deduplicate `name` and `self._name` but I agree that it makes the diff unnecessarily complicated.